recipe/session/refresh.go

```diff
@@ -13,7 +13,7 @@
  * under the License.
  */
 
-package api
+package session
```

Comment:
Yep, removed the internal api directory to remove a circular import issue (fixing it correctly would require a massive refactor which I wanted to avoid)